tqa-web-app/src/app/app-routing.module.ts

```diff
@@ -1,9 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { AdminAuthComponent } from './components/admin-auth/admin-auth.component';
+import { AdminComponent } from './components/admin/admin.component';
 import { MainComponent } from "./components/main/main.component";
 
 const routes: Routes = [
   { path: '', component: MainComponent },
+  { path: 'admin-auth', component: AdminAuthComponent },
```

Comment:
We will probably want to create a constants file that contains all of these routes as well and then reference the routes via those constants.  That way it will be easy to change or reference the exact same path in other components during navigation.

Reply:
path name change. I think the const routes: Routes serve this purpose well